Makefile

```diff
@@ -185,107 +184,244 @@ test-integration: test-integration-aws test-integration-clickhouse test-integrat
 test-integration: test-integration-gcp test-integration-influxdb test-integration-kafka test-integration-loki
 test-integration: test-integration-pulsar test-integration-splunk
 
+stop-test-integration: ## Stops all integration test infrastructure
```

Comment:
Ahhh... I sent for `start-integration` and `stop-integration` since there was `start-integration-aws` etc... Do you prefer this way? Or maybe it was just a bit of inconsistency?

Reply:
I just prefixed test-integration with stop-. I wanted to leave the original step name along as I figured people would already know it. This makes it a non-breaking change. Do you think it works better the other pattern?

Reply:
I think it's fine. :) I was just curious.